### ISSUE TITLE
flip function results in mojibake for wide characters in narrow Python builds

### ISSUE THREAD

**Issue by mistydemeo** [mistydemeo/xenoglossia#8]
The `flip` function produces mojibake if the input string contains wide characters, when using Xenogossia on narrow Python builds.

Expected (as seen in Python 3):

``` xg
$ echo 'test😌' | xg 'flip' | pbcopy
（╯°□°）╯︵ 😌ʇsǝʇ
```

Actual (as seen in a narrow build of Python 2.7):

``` xg
# output is completely broken when piping into another program
$ echo 'test😌' | xg 'flip' | pbcopy
Ôºà‚ïØ¬∞‚ñ°¬∞Ôºâ‚ïØÔ∏µ Ì∏åÌ†Ω ás«ù á
# results are easier to read when stdout has an encoding
$ echo 'test😌' | xg 'flip'
（╯°□°）╯︵ ������ʇsǝʇ
```

**Comment by mistydemeo** [comment]
This is happening because narrow builds of Python 2 consider astral plane characters, like emoji, to be two characters and consequently splits them inappropriately during the reverse.

**Comment by mistydemeo** [comment]
Fixed by #9.
